.github/workflows/generate-build-badges.yml

```
@@ -0,0 +1,28 @@
+name: Generate Build Badges
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    branches: develop
+    types:
+      - completed
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - id: extract
+        run: |
+          data=$(gh api /repos/DARMA-tasking/vt/actions/runs/$RUN_ID/jobs)
```

Optional: can we drop `RUN_ID` as a separate variable? It's only used once.

a.k.a. does this work? 
```suggestion
          data=$(gh api /repos/DARMA-tasking/vt/actions/runs/${{ github.event.workflow_run.id }}/jobs)
```

---

Yes, this should work